packages/map-components/templates/webpack/package.json

```diff
@@ -6,9 +6,9 @@
     "start": "webpack-dev-server --mode development --open"
   },
   "dependencies": {
-    "@arcgis/core": "~4.28.0",
-    "@arcgis/map-components": ">=4.28.0-beta.39 < 4.29",
-    "@esri/calcite-components": "^1.9.2"
+    "@arcgis/core": ">=4.29.0-next.20240207",
+    "@arcgis/map-components": ">=4.29.0-beta.100 < 4.30",
+    "@esri/calcite-components": "^2.4.0"
   },
   "devDependencies": {
     "css-loader": "^6.7.2",
```

Comment:
Check dep for latest, I think css-loader is 6.10.0. Some of these libraries sneak in breaking changes, etc in minor releases that should always be verified.